### PR TITLE
Create 4.11.0.md

### PR DESCRIPTION
**Pull request by vlmakarov**

### CODE REVIEW

File: _docs/release-notes/basespace/2016/4.11.0.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+title: September 6, 2016 - 4.11.0 - End of Free Trial Period
+section_root: release-notes
+category: release notes
+tags: [basespace]
+version: 4.11.0
+release_date: 2016-09-06
+---
+
+## What's changed for BaseSpace 4.11.0
+- End of free trial period for existing users and introduction of commercial restrictions
```
Comment (amaynard) [inline _docs/release-notes/basespace/2016/4.11.0.md]: What are the restrictions? Suggest
-- Apps that incur a compute charge are not available for basic users, with a few exceptions including FASTQ Generation.

File: _docs/release-notes/basespace/2016/4.11.0.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+- Enabled manual launch for MiSeq users and dropped MSR workflow
```
Comment (amaynard) [inline _docs/release-notes/basespace/2016/4.11.0.md]: Manual launch sounds bad and is wrong. Should also refer to support note on this.

File: _docs/release-notes/basespace/2016/4.11.0.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+- Enhanced user experience when transferring and sharing data with non-BSSH users
```
Comment (amaynard) [inline _docs/release-notes/basespace/2016/4.11.0.md]: change non BSSH users to people not currently using BaseSpace SequenceHub.

File: _docs/release-notes/basespace/2016/4.11.0.md (hunk 1 above)
Comment (vlmakarov) [inline _docs/release-notes/basespace/2016/4.11.0.md]: Sounds negative (although is factually true); for that reason, I'd rather not say it

File: _docs/release-notes/basespace/2016/4.11.0.md (hunk 2 above)
Comment (vlmakarov) [inline _docs/release-notes/basespace/2016/4.11.0.md]: Please suggest an alternative wording and a pointer to the support note (I cannot find any)

**Pull request closed without merging**